judgments/utils.py

```diff
@@ -212,4 +212,4 @@ def get_document_by_uri(document_uri: str) -> Judgment:
     )
 
     # raises a JudgmentNotFoundError if it doesn't exist
-    return Judgment(document_uri, api_client)
+    return api_client.get_document_by_uri(document_uri)
```

Comment:
looks like we don't currently test
`get_document_by_uri` and we mock it out in any test of functions that call it.

Can we add the following tests for it:


```
import unittest
from judgments.utils import get_document_by_uri

class TestGetDocumentByUriOrRaise(unittest.TestCase):
    @mock.patch('judgments.utils.MarklogicApiClient')
    def test_get_existing_document(self, mock_api_client):

        document = Mock()
        mock_api_client.return_value.get_document_by_uri.return_value = document

        result = get_document_by_uri_or_raise('sample_uri')

        self.assertEqual(result, document)

    @mock.patch('judgments.utils.MarklogicApiClient')
    def test_get_nonexistent_document(self, mock_api_client):
        mock_api_client.return_value.get_document_by_uri.side_effect = DocumentNotFoundError

        with self.assertRaises(DocumentNotFoundError):
            get_document_by_uri_or_raise('nonexistent_uri')
```